app/build.gradle

```diff
@@ -42,9 +42,21 @@ android {
     productFlavors {
         pdd {
             applicationId 'org.stepik.android.adaptive.pdd'
-            signingConfig signingConfigs.release
-            versionCode 6
-            versionName "1.3"
+        }
+        gmat1906 {
```

Comment:
use `applicationIdSuffix ` https://developer.android.com/studio/build/build-variants.html instead of full name of package